### ISSUE TITLE
Feedback #3

### ISSUE THREAD

**Issue by bursteinalan** [CS521-Burstein/final-project-yupj007#2]
- NA
- Why are there three notebooks? You should all be working together in one notebook, using Git to collaborate

**Comment by chaosx1102** [comment]
Yes professor, we just make it wrong. Cause one teammate upload it and another one download it and upload again with his name.
And finally, we integrate three files into one ipynb file
